### PR TITLE
nbd: unmap wait process exit

### PR DESCRIPTION
**Pull request by wu-hanqing**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Review by bai-charisu — APPROVED** [review]

File: nbd/src/NBDTool.cpp [hunk 1]
```diff
@@ -150,5 +170,31 @@ ImagePtr NBDTool::GenerateImage(const std::string& imageName) {
     return result;
 }
 
+int NBDTool::WaitForTerminate(const std::string& devpath, pid_t pid,
+                              uint64_t totalRetryMs) {
+    if (pid < 0) {
+        return 0;
+    }
+
+    // TODO(wuhanqing): make this to NBDConfig
```
Comment (xu-chaojie) [inline nbd/src/NBDTool.cpp:179]: 这个TODO处理掉吧，没必要留着，考虑下需不需要加到配置里去，如果不需要，就单独一个常量定义就可以了，如果需要就增加到配置里吧

File: nbd/src/NBDTool.cpp [hunk 2]
```diff
@@ -111,8 +111,26 @@ int NBDTool::Connect(NBDConfig *cfg) {
...
+    }
+
     NBDControllerPtr nbdCtrl = GetController(false);
-    return nbdCtrl->DisconnectByPath(devpath);
+    ret = nbdCtrl->DisconnectByPath(devpath);
+    if (ret != 0) {
+        return ret;
+    }
+
+    ret = WaitForTerminate(devpath, devpid, 5000);
```
Comment (xu-chaojie) [inline nbd/src/NBDTool.cpp:131]: 这个5000也考虑下是否需要加到配置中，或者定义成常量，不要用裸的数字。

Comment (wu-hanqing) [reply nbd/src/NBDTool.cpp]: Done

File: nbd/src/NBDTool.cpp (hunk 1 above)
Comment (wu-hanqing) [reply nbd/src/NBDTool.cpp]: Done

**Review by xu-chaojie — APPROVED** [review]

**Pull request merged**